server/midddleware/auth.go

```diff
@@ -100,7 +101,16 @@ func AuthFunction(ctx context.Context, jwtValidator *validator.Validator, config
 			}
 		}
 	}()
-
+	// disable health check authn/z
+	var fullMethodName string
+	fullMethodNameFound := false
+	clientCtx := inprocgrpc.ClientContext(ctx)
+	if clientCtx != nil {
+		fullMethodName, fullMethodNameFound = runtime.RPCMethod(clientCtx)
+	}
+	if fullMethodNameFound && fullMethodName == "/.HealthAPI/Health" {
```

Comment:
Should we have a list for this? Or const string defined above if it is just one.

Reply:
sounds good. fixed.